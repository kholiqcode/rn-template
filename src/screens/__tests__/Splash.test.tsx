import React from 'react';
import { render } from '@testing-library/react-native';
import Splash from '../Splash';
import timeTravel, { setupTimeTravel } from '../../../__mocks__/timeTravel';

beforeEach(setupTimeTravel);

describe('Splash screen', () => {
  it('Render Splash screen', () => {
    const splash = render(<Splash />).toJSON();
    timeTravel(500);
    expect(splash).toBeTruthy();
  });
});
