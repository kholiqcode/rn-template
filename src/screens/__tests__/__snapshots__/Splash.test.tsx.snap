// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash screen Render Splash screen 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    collapsable={false}
    source={
      Object {
        "testUri": "../../../src/assets/images/common/logo.png",
      }
    }
    style={
      Object {
        "height": 300,
        "transform": Array [
          Object {
            "rotate": "0deg",
          },
        ],
        "width": 300,
      }
    }
  />
</RCTSafeAreaView>
`;
